### PR TITLE
COP-8583 Add COP ref UI to confirmation page

### PR DESCRIPTION
**Pull request by Ali-Baker**
Ticket: https://support.cop.homeoffice.gov.uk/browse/COP-8583

In this PR
- Get the COP reference number from keycloak and pass it to the confirmation page
- Display COP reference number on panel in confirmation page

# How to test
- Reset the active flag from keycloak
- Visit the COP home page
- Register as new user

## Expected behaviour
- Once registered you should be take to the `Confirmation page`
- You should expect to see the `reference` number
- The page layout should match the [Figma design](https://www.figma.com/file/HvHakMNbQ9Z17Arw8flhuy/COP?node-id=1875%3A1713)

image

### CODE REVIEW

File: client/src/pages/forms/Confirmation.jsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
     <>
       <div className="govuk-panel govuk-panel--confirmation">
         <h1 className="govuk-panel__title">Account created</h1>
-        <div className="govuk-panel__body">We have sent you a confirmation email</div>
+        <div className="govuk-panel__body">COP reference:</div>
+        <div className="govuk-panel__body">
+          <strong>{reference}</strong>
+        </div>
       </div>
+      <p className="govuk-body">We have sent you a confirmation email</p>
```
Comment (paulfgraham) [inline client/src/pages/forms/Confirmation.jsx:15]: The screenshot includes a full stop on the end of this sentence:
```suggestion
      <p className="govuk-body">We have sent you a confirmation email.</p>
```

**Review by paulfgraham — CHANGES_REQUESTED** [review]
One minor tweak requested, based on the screenshot - a trailing full stop.

Are there any unit tests for this? If so, I assume they need changing; if not, there probably ought to be...

**Comment by Ali-Baker** [comment]
> One minor tweak requested, based on the screenshot - a trailing full stop.
> 
> Are there any unit tests for this? If so, I assume they need changing; if not, there probably ought to be...

There're for routing for this page I made on previous PR. Wasn't sure about this small change. Snapshot testing maybe.

**Review by paulfgraham — APPROVED** [review]
Thanks for making the requested change and answering my query about the tests, @Ali-Baker. Happy to approve this now. 👍

**Review by jennikate — APPROVED** [review]

**Pull request merged**